2023-01-17 17:44:12,178: [12] m.main:INFO: Registered plugins: IntrospectionAPIPlugin, <class 'manticore.ethereum.plugins.SkipRevertBasicBlocks'>, <class 'manticore.ethereum.plugins.FilterFunctions'>
2023-01-17 17:44:12,179: [12] m.main:INFO: Beginning analysis
2023-01-17 17:44:12,183: [12] m.e.manticore:INFO: Starting symbolic create contract
2023-01-17 17:50:24,609: [12] m.e.manticore:INFO: Found a concrete globalsha3 b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00' -> 32434096268151414502930714319429211172703610272722345406708838912687411675598
2023-01-17 17:50:27,934: [12] m.e.manticore:INFO: Starting symbolic transaction: 0
2023-01-17 17:56:13,466: [12] m.c.worker:ERROR: Exception in state 5: ManticoreError('Forking on unfeasible constraint set')
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/worker.py", line 137, in run
    current_state.execute()
  File "/usr/local/lib/python3.7/dist-packages/manticore/ethereum/state.py", line 8, in execute
    return self._platform.execute()
  File "/usr/local/lib/python3.7/dist-packages/manticore/platforms/evm.py", line 3106, in execute
    self.current_vm.execute()
  File "/usr/local/lib/python3.7/dist-packages/manticore/platforms/evm.py", line 1322, in execute
    raise Concretize("Symbolic PC", expression=expression, setstate=setstate, policy="ALL")
manticore.core.state.Concretize

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/worker.py", line 158, in run
    m._fork(current_state, exc.expression, exc.policy, exc.setstate, exc.values)
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/manticore.py", line 518, in _fork
    raise ManticoreError("Forking on unfeasible constraint set")
manticore.exceptions.ManticoreError: Forking on unfeasible constraint set
 
2023-01-17 17:57:41,124: [12] m.e.manticore:INFO: Found a concrete globalsha3 b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00' -> 73138091548938366139099779971772605061518111956507126412222883840269740007539
2023-01-17 18:00:18,132: [12] m.e.manticore:INFO: Found a concrete globalsha3 b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01' -> 27376155845717860833342594873357525089589924219251518322086369159840883582034
2023-01-17 18:06:09,939: [12] m.e.manticore:INFO: 3 alive states, 1 terminated states
2023-01-17 18:09:42,556: [82] m.c.plugin:WARNING: Caught will_solve in state None, but failed to capture its initialization
2023-01-17 18:09:42,953: [81] m.c.plugin:WARNING: Caught will_solve in state None, but failed to capture its initialization
2023-01-17 18:09:43,822: [82] m.c.manticore:INFO: Generated testcase No. 0 - RETURN(2 txs)
2023-01-17 18:09:44,453: [81] m.c.manticore:INFO: Generated testcase No. 1 - RETURN(2 txs)
2023-01-17 18:10:32,704: [80] m.c.plugin:WARNING: Caught will_solve in state None, but failed to capture its initialization
2023-01-17 18:11:02,067: [80] m.c.manticore:INFO: Generated testcase No. 2 - RETURN(2 txs)
